Userland/Libraries/LibWeb/MimeSniff/Resource.cpp

```diff
@@ -7,6 +7,256 @@
 #include <LibWeb/Fetch/Infrastructure/URL.h>
 #include <LibWeb/MimeSniff/Resource.h>
 
+namespace {
+
+using namespace Web::MimeSniff;
+using Byte = u8;
+
+class BytePatternTableRow {
```

Comment:
This class doesn't seem to have any invariants: It just holds data. As such, it should probably just be a "plain old data" struct, with all public data members.